pathways/shipments.py

```diff
@@ -479,53 +479,78 @@ def add_pest_clusters(config, shipment):
         cluster_sizes = _infested_stems_to_cluster_sizes(
             infested_stems, max_infested_stems_per_cluster
         )
-        for cluster_size in cluster_sizes:
-            # Generate cluster as indices in the array of stems
-            distribution = config["clustered"]["distribution"]
-            if distribution == "gamma":
-                param1, param2 = config["clustered"]["parameters"]
+        cluster_indexes = []
+        distribution = config["clustered"]["distribution"]
+        if distribution == "gamma":
+            param1, param2 = config["clustered"]["parameters"]
+            gamma_min_max = stats.gamma.interval(0.999, param1, scale=param2)
```

Comment:
The best practice would be to assign 0.999 to a variable as a basic form of documentation. (Using keyword arguments alpha=0.999 is not enough since that does not make it more clear.)